core/flow/traffic_shaping_test.go

```diff
@@ -0,0 +1,142 @@
+package flow
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	"github.com/sentinel-group/sentinel-golang/core/base"
```

Comment:
Please import "github.com/alibaba/sentinel-golang/core/base"
the module "github.com/sentinel-group/sentinel-golang/" had been deprecated.

Reply:
![image](https://user-images.githubusercontent.com/6278792/75434006-dbbacc80-598b-11ea-924f-ced455aa54d8.png)

Reply:
Please run rebase from github.com/alibaba/sentinel-golang master before test.

Reply:
ok
package rename